web/layouts/nav.html.erb

```diff
@@ -0,0 +1,97 @@
+<div id="wrap">
+  <nav>
+    <img class="logo" src="/img/helmet.svg" />
+    <button type="button" class="btn-hamburger" data-action="nav-toggle">
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+    </button>
+    <ul class="nav-menu">
+      <li class="nav-item"><a href="#">Home</a></li>
+      </li>
+      <li class="nav-item dropdown">
+        <a href="#" data-action="dropdown-toggle">Foreman Stable (3.1)</a>
+        <div class="dropdown-menu">
+          <a class="dropdown-item" href="#">Guide 1</a>
+          <a class="dropdown-item" href="#">Guide 2</a>
+          <a class="dropdown-item" href="#">Guide 3</a>
+        </div>
+      </li>
+      <li class="nav-item dropdown">
+        <a href="#" data-action="dropdown-toggle">Foreman 3.0</a>
+        <div class="dropdown-menu">
+          <a class="dropdown-item" href="#">Guide 1</a>
+          <a class="dropdown-item" href="#">Guide 2</a>
+          <a class="dropdown-item" href="#">Guide 3</a>
+        </div>
+      </li>
+      <li class="nav-item dropdown">
+        <a href="#" data-action="dropdown-toggle">Nightly</a>
+        <div class="dropdown-menu">
+          <a class="dropdown-item" href="#">Guide 1</a>
+          <a class="dropdown-item" href="#">Guide 2</a>
+          <a class="dropdown-item" href="#">Guide 3</a>
+        </div>
+      </li>
+    </ul>
+  </nav>
+</div>
+<script>
+let nav = document.querySelector("nav");
+let dropdown = nav.querySelector(".dropdown");
+let dropdownToggle = nav.querySelector("[data-action='dropdown-toggle']");
+let navToggle = nav.querySelector("[data-action='nav-toggle']");
+
+dropdownToggle.addEventListener("click", () => {
+  if (dropdown.classList.contains("show")) {
+    dropdown.classList.remove("show");
+  } else {
+    dropdown.classList.add("show");
+  }
+});
+
+navToggle.addEventListener("click", () => {
+  if (nav.classList.contains("opened")) {
+    nav.classList.remove("opened");
+  } else {
+    nav.classList.add("opened");
+  }
+});
+
+// TODO: load menu dynamically from an external JSON file
+var menu = JSON.parse(`
+{
+  "menu": {
+    "Foreman Stable (3.1)": {
+      "Planning for Foreman": "#",
+      "Foreman Quickstart Guide on RHEL/CentOS
+": "#",
+      "Foreman Quickstart Guide on Debian/Ubuntu": "#",
+      "Installing Foreman on RHEL/CentOS": "#",
+      "Installing Smart Proxy with Content on RHEL/CentOS
+": "#",
+      "Installing Smart Proxy on RHEL/CentOS": "#",
+      "Installing Foreman on Debian/Ubuntu": "#",
+      "Installing Smart Proxy on Debian/Ubuntu": "#",
+      "Upgrading and Updating Foreman": "#",
+      "Deploying Foreman on AWS": "#",
+      "Configuring Smart Proxies with a Load Balancer": "#",
+      "Provisioning Guide": "#",
+      "Configuring Foreman to use Ansible": "#",
+      "Managing Hosts Guide": "#",
+      "Administering Foreman": "#",
+      "Application Centric Deployment": "#"
+    },
+    "Katello Stable (4.0)": {
```

Comment:
Latest Katello stable is 4.2 (which maps to Foreman 3.0)

Reply:
Fixed.